src/gtk/webview_webkit.cpp

```diff
@@ -949,10 +949,12 @@ wxString wxWebViewWebKit::GetPageText() const
                     wxConvUTF8);
 }
 
-void wxWebViewWebKit::RunScript(const wxString& javascript)
+bool wxWebViewWebKit::RunScript(const wxString& javascript, wxString* output)
 {
     webkit_web_view_execute_script(m_web_view,
                                    javascript.mb_str(wxConvUTF8));
+
+    return (output == NULL);
```

Comment:
I don't think this is correct... So if we use some JS code with non-null output, the JS is still getting executed but then we return false and the program believes there was an error with executing it.

IMO it would be better to check for `output != NULL` first and return false _without_ executing JS in this case.